packages/ui/__tests__/Checkbox.test.tsx

```diff
@@ -0,0 +1,104 @@
+    const onChange = jest.fn()
+    const wrapper = await mountAndCheckA11Y(<Checkbox checked name="hello" onChange={onChange} label="Hello World" />)
+
+    expect(wrapper.find(Check).exists()).toBeTruthy()
+    expect(wrapper.find(Minus).exists()).toBeFalsy()
+    expect(wrapper.find('input').getDOMNode<HTMLInputElement>().indeterminate).toBeFalsy()
+  })
+
+  it('Renders a checkbox, checks that properties are passed to an underlying input', async () => {
+    const onChange = jest.fn()
+    const onBlur = jest.fn()
+    const wrapper = await mountAndCheckA11Y(
+      <Checkbox checked name="hello" disabled required value="world" onChange={onChange} onBlur={onBlur} label="Hello World" />,
+    )
+    expect(wrapper.find('input').props()).toEqual({
+      type: 'checkbox',
+      name: 'hello',
+      value: 'world',
+      onChange,
+      onBlur,
+      checked: true,
+      'aria-invalid': false,
+      'aria-required': true,
+      'aria-describedby': undefined,
+      'aria-errormessage': undefined,
+      disabled: true,
+      id: 'uuidtest',
+    })
+  })
+
+  it('Renders an invalid checkbox, checks that properties are passed to an underlying input', async () => {
+    const onChange = jest.fn()
+    const onBlur = jest.fn()
+    const wrapper = await mountAndCheckA11Y(
+      <Checkbox checked error="test" name="hello" disabled value="world" onChange={onChange} onBlur={onBlur} label="Hello World" />,
+    )
+    expect(wrapper.find('input').props()).toEqual({
+      type: 'checkbox',
+      name: 'hello',
+      value: 'world',
+      onChange,
+      onBlur,
+      checked: true,
+      'aria-invalid': true,
+      'aria-required': undefined,
+      'aria-describedby': undefined,
+      'aria-errormessage': errorId('uuidtest'),
+      disabled: true,
+      id: 'uuidtest',
+    })
+  })
+
+  it('Renders the indeterminate checkbox', async () => {
+    const onChange = jest.fn()
+    const wrapper = await mountAndCheckA11Y(<Checkbox checked indeterminate name="hello" onChange={onChange} label="Hello World" />)
+
+    expect(wrapper.find(Check).exists()).toBeFalsy()
+    expect(wrapper.find(Minus).exists()).toBeTruthy()
+    expect(wrapper.find('input').getDOMNode<HTMLInputElement>().indeterminate).toBe(true)
+  })
+
+  it('Checkbox is passed a disabled property, input contains disabled property', async () => {
+    const onChange = jest.fn()
+    const wrapper = await mountAndCheckA11Y(
+      <Checkbox checked indeterminate name="hello" disabled onChange={onChange} label="Hello World" />,
+    )
+
+    expect(wrapper.find('input').getDOMNode<HTMLInputElement>().disabled).toBe(true)
+  })
+
+  it('Checkbox has been passed an error property, error message is displayed', async () => {
+    const onChange = jest.fn()
+    const wrapper = await mountAndCheckA11Y(<Checkbox name="hello" onChange={onChange} label="Hello World" error="Unexpected Error" />)
+
+    expect(wrapper.find(Error).exists()).toBe(true)
```

Comment:
```suggestion
    expect(wrapper.find(Error).exists()).toBeTruthy()
```

Reply:
I check for the explicit true value which is returned by `exists()`, I see nothing wrong about it. It's more explicit imo